### ISSUE TITLE
Add DEPLOY_TOKEN

### ISSUE THREAD

**Issue by bastimeyer** [streamlink/streamlink-twitch-gui-wiki#1]
@beardypig, not sure if you've missed my messages on Gitter, but since you're the only one with access to the @streamlink-bot account, I need you to create a personal access token for the bot and add it here to the secret env vars as `DEPLOY_TOKEN`, so that it can automatically push the wiki changes from this mirror-repo to the actual wiki repo.

Thanks!

- https://github.com/streamlink/streamlink-twitch-gui-wiki/blob/4319cf6f8cb82fa641b24bc0944b05eda5dc433e/.github/workflows/main.yml#L16
- https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token

**Comment by beardypig** [comment]
OK done, lemme know if it works or if I messed it up :)

**Comment by bastimeyer** [comment]
Thanks!

It doesn't work for some reason and I'm not sure why:
https://github.com/streamlink/streamlink-twitch-gui-wiki/runs/1500157576?check_suite_focus=true#step:3:9

> remote: Permission to streamlink/streamlink-twitch-gui.wiki.git denied to github-actions[bot].
> fatal: unable to access 'streamlink/streamlink-twitch-gui.wiki.git': The requested URL returned error: 403

----

I'll try it again with this change, but if this doesn't work either, I don't know what's wrong. The bot has write access to this repo (probably unnecessary) and the Twitch GUI repo where the wiki option is set to write-access only.
```diff
-git push "https://${DEPLOY_TOKEN}@github.com/${GITHUB_REPOSITORY%-wiki}.wiki.git"
+git push "https://streamlink-bot:${DEPLOY_TOKEN}@github.com/${GITHUB_REPOSITORY%-wiki}.wiki.git"
```

**Comment by bastimeyer** [comment]
https://github.com/streamlink/streamlink-twitch-gui-wiki/runs/1500250102?check_suite_focus=true#step:3:8

**Comment by bastimeyer** [comment]
Fixed via 5d8a96f80dded55e8f6bd84db0ecb35f3f22e42f and 4d8faf10a8c6276338d85f8a37f90b29eb3f55d0

`actions/checkout@v2` automatically sets an auth HTTP header to the git config of the cloned repo, which gets sent with all https pushes, which is not what we want if we want to push to a different repo with a different token.
https://github.com/actions/checkout/blob/5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f/src/git-auth-helper.ts#L55-L59
